<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="3" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lb_an">
        <property name="text">
         <string>点击目标:(这里可以添加N个按钮目标,只要网页里出现了它,就会帮忙点击)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_sub">
        <property name="text">
         <string>删除一行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_add">
        <property name="text">
         <string>添加一行</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5" rowspan="6">
     <widget class="QTextEdit" name="ed_log">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>99999</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;日志:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请注意必须先设置好所有东西然后再去增加浏览器,不然设置会无效!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QTableWidget" name="tab_mban">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;这里请注意了: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;如果是要输入 注意前缀格式&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#0000ff;&quot;&gt;$$&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;比如:&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#0000ff;&quot;&gt; $$123456 &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;那它就会输入&lt;/span&gt;&lt;span style=&quot; font-size:11pt; color:#ffaa00;&quot;&gt;123456&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;注意这里如果是填的输入,那CSS就必须设置&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;进阶玩法:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;这里的目标按钮支持前缀%% 来控制不重复点击&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;1.一开始是灰色的按钮,或者点击后马上就跳转到下一个页面的就不需要前缀%%&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;2.如果是随时就可以点击成功的按钮,则在前面加入%%&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;例如:&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#0000ff;&quot;&gt;%%加入购物车&lt;/span&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ffaa00;&quot;&gt;%%预约&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;原理解析:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;因为正常情况下,程序是会不断的去点击目标按钮的直到整个按钮不见了,程序才会往下走,去判断下一个步骤,比如淘宝的立即购买,一开始是没有这个按钮的,或者是点不动的,只有到时间了才能点击,点击后,直接是调到提交订单的页面了,也就是这个立即购买按钮已经消失了,所以程序会执行下一步,而这个%%就是不判断是否点击成功,反正就找到这个按钮了,就帮忙点击1次,没找到就等待它出现.点完后就开始下一个步骤!仔细理解一下就明白区别了!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>目标按钮/输入框</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>目标CSS选择器</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>天猫</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>立即购买</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>#J_LinkBuy</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>天猫</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>%%提交订单</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>#submitOrderPC_1 &gt; div &gt; a</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>天猫输入密码</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>$$123456</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>#payPassword_rsainput</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>天猫</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>%%确认付款</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>#J_authSubmit</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QTextEdit" name="te_help">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600;&quot;&gt;使用步骤: &lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600; color:#ffaa00;&quot;&gt;欢迎在论坛留言分享你的配置&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;演示视频&lt;/span&gt;&lt;a href=&quot;https://www.bilibili.com/video/BV1S54y147xc&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://www.bilibili.com/video/BV1S54y147xc&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;网友分享&lt;/span&gt;&lt;a href=&quot;https://lanren.lanzous.com/b00u4meta &quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://lanren.lanzous.com/b00u4meta &lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:12pt; color:#0000ff;&quot;&gt;密码:lanren&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600; color:#0000ff;&quot;&gt;本工具依然是看脸,不过可以比普通人同时多看几次脸&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;1. 点击[&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600; color:#ff0000;&quot;&gt;增加浏览器&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;]就是多增加一个浏览器&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;2. 在浏览器中登录你的账号,并把产品的尺寸,规格,型号等参数手动弄好!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;3. 设置好一切后,点击[&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600; color:#ff0000;&quot;&gt;全部开始&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;]&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600; color:#ff0000;&quot;&gt;为了让大家能理解,给大家说下软件的工作原理:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;1. 原理就是多开浏览器,然后脚本同步进行控制&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;2. 当点击全部开始后它就会根据你填写的目标按钮以及Css选择器作为定位目标按钮的依据&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;3. 优先CSS选择器(下方有教程),如果没找到,则会通过目标按钮名来定位目标按钮&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;4. 只需要满足其中一个条件就可以找到目标,然后就会不断点击&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;5.支持判断N个按钮,比如你的购买步骤比较多时,只要提前加入好目标按钮名和选择器就可以骚气运行&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;6.支持帮忙输入文字的操作,比如帮忙输入密码&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;7.它是按照顺序往下走的(注意了我特意改进了一下)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600;&quot;&gt;如何填写目标按钮 和 CSS选择器呢?&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600;&quot;&gt;目标名:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;目标按钮名,很容易理解就是你需要帮忙点击的按钮的标题(必须是可以开始抢购时的按钮名)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;如果是需要输入文字,则这里的填写格式为$$文字 比如:$$123456 它就会输入123456&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; color:#ff0000;&quot;&gt;如果是选择帮忙输入了,那就只能用CSS选择器的方式定位输入框&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:600; color:#0000ff;&quot;&gt;进阶玩法:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:11pt; color:#ff0000;&quot;&gt;这里的目标按钮支持前缀%% 来控制不重复点击 ##来配置下拉滚动条 可以同时存在&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;1.一开始是灰色的按钮,或者点击后马上就跳转到下一个页面的就不需要前缀%%&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;2.如果是随时就可以点击成功的按钮,则在前面加入%%&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;例如:&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;%%加入购物车 &lt;/span&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;%%预约&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; color:#000000;&quot;&gt;3.如果跳转后目标按钮并没有第一时间渲染显示,需要下划滚动条,则在目标按钮名字中随便一个位置加入##来标识&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;例如&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑'; color:#000000;&quot;&gt;:京东的提交订单 可以这样填:&lt;/span&gt;&lt;span style=&quot; font-family:'微软雅黑'; color:#ff0000;&quot;&gt;##提交订单&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; color:#ff0000;&quot;&gt;原理解析:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;因为正常情况下,程序是会不断的去点击目标按钮的直到整个按钮不见了,程序才会往下走,去判断下一个步骤,比如淘宝的立即购买,一开始是没有这个按钮的,或者是点不动的,只有到时间了才能点击,点击后,直接是调到提交订单的页面了,也就是这个立即购买按钮已经消失了,所以程序会执行下一步,而这个%%就是不判断是否点击成功,反正就找到这个按钮了,就帮忙点击1次,没找到就等待它出现.点完后就开始下一个步骤!仔细理解一下就明白区别了!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-weight:600;&quot;&gt;目标CSS选择器:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;(1) 首先用Google Chrome浏览器打开你的商品网站&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;(2) 然后按下F12进入开发者模式&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;(3) 然后看图操作&lt;/span&gt;&lt;a href=&quot;https://attach.52pojie.cn//forum/202011/12/165648ml2jxid0bzflaz0f.gif&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://attach.52pojie.cn//forum/202011/12/165648ml2jxid0bzflaz0f.gif&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="uRLLabel">
        <property name="text">
         <string>商品链接:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ed_url">
        <property name="text">
         <string>https://www.vmall.com/product/10086726905036.html</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>启动时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dte_time">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;注意了,这里的时间是本地电脑的时间&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;提前刷新是会在时间到的时候先刷新然后再开始&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;(如果太晚了,可以时间稍微提前一点)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss.zzz</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_sfds">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;注意了,这里的时间是本地电脑的时间&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;提前刷新是会在时间到的时候先刷新然后再开始&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;(如果太晚了,可以时间稍微提前一点)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>定时启动</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_tqsx">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;注意了,这里的时间是本地电脑的时间&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;提前刷新是会在时间到的时候先刷新然后再开始&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;(如果太晚了,可以时间稍微提前一点)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>提前刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_openWeb">
        <property name="text">
         <string>1.增加浏览器</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_start">
        <property name="text">
         <string>2.全部开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_close_all">
        <property name="text">
         <string>3.关闭所有浏览器</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>tab_mban</zorder>
   <zorder>ed_log</zorder>
   <zorder>te_help</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
